core/arch/aarch64/aarch64.asm

```diff
@@ -627,197 +626,55 @@ GLOBAL_LABEL(tlsdesc_resolver:)
         ret
 
 /* This function is called from the fragment cache when the original code had
- * IC IVAU, Xt. Typically it just records which cache lines have been invalidated
- * and sets icache_op_struct.flag. However, if non-contiguous cache lines have
- * been invalidated we branch to fcache_return instead of returning.
- * When we enter here:
+ * IC IVAU, Xt. When we enter here:
  *
  * X0 contains the pointer to spill_state_t.
- * X30 contains the return address in the fragment cache.
+ * X30 contains the return address in the fragment cache (no longer required).
```

Comment:
Please update the mangle code to use a plain indirect branch instead of branch and link and then remove this comment. Otherwise it will be confusing to look at the mangle code and wonder why it does a branch and link with code after it. Someone is going to think there is an error.